packages/api/lib/request.js

```diff
@@ -35,6 +41,18 @@ async function verifyJwtAuthorization(requestJwtToken) {
   return accessToken;
 }
 
+/**
+* Validate request has header matching expected minimum version
+* @param {Request} req - express Request object
+* @param {number} minVersion - Minimum API version to allow
+* @returns {boolean}
+*/
+function isMinVersionApi(req, minVersion) {
+  const requestVersion = Number(req.headers.version);
+  if (requestVersion && minVersion <= requestVersion) return true;
+  return false;
```

Comment:
```suggestion
  return requestVersion && minVersion <= requestVersion;
```

Reply:
That actually doesn't *really* work  because of the point you brought up during the CR - 

`requestVersion && minVersion <= requestVersion;` evaluates type to `number | boolean` via ts-check when you pull it out of the if() expression *because* zero is falsy and will return zero.... which isn't a boolean.

As always, TS is correct -- whereas this 'works' with it in the if() expression, outside of it, not so much.   

And better to be explicit anyway: 

7792475cc4

Reply:
7792475cc4